terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/scale-down.test.ts

```diff
@@ -1265,6 +1386,122 @@ describe('scale-down', () => {
     });
   });
 
+  describe('cleanupOldSSMParameters', () => {
+    it('Stops when LastModifiedDate === undefined', async () => {
```

Comment:
Well, this is odd, but in the AWS docs I don't find it to be possible to ever not be defined. But, in the interface description on typescript it can.

So I decided to properly treat this case, for the sake of code completeness / safeness. And decided to consider it as soo new that this value is yet to be populated.

Random guess.